### ISSUE TITLE
what is the right way to add rxjs operator? such as exhaustMap

### ISSUE THREAD

**Issue by goldenbearkin** [mgechev/angular2-seed#784]
as subject. Anybody can help? i have been solving this for two days. Many thanks!

**Comment by mgechev** [comment]
`import "rxjs/add/operator/exhaustMap";`

**Comment by goldenbearkin** [comment]
i have already tried it, still doesnt work. Do a need to config the project.config.ts / see.config.ts?

**Comment by mgechev** [comment]
No, you don't. Take a look at `rxjs/add/operator` and see if there's such file, ending with .js.

**Comment by goldenbearkin** [comment]
yes. it has. but the broswer keeps yelling GET http://localhost:5555/rxjs/add/operator/exhaustMap.js 404 (Not Found)

**Comment by mgechev** [comment]
I have a bunch of operators imported this way in my project:

```ts
import 'rxjs/add/operator/filter';
import 'rxjs/add/operator/map';
import 'rxjs/add/operator/combineAll';
import 'rxjs/add/operator/takeLast';

```

**Comment by goldenbearkin** [comment]
```javascript
protected SYSTEM_CONFIG_DEV = {
        defaultJSExtensions: true,
        packageConfigPaths: [`${this.APP_BASE}node_modules/*/package.json`, `${this.APP_BASE}node_modules/@angular2-material/*/package.json`],
        paths: {
            [this.BOOTSTRAP_MODULE]: `${this.APP_BASE}${this.BOOTSTRAP_MODULE}`,
            'angular2/*': `${this.APP_BASE}angular2/*`,
            // 'rxjs/*': `${this.APP_BASE}rxjs/*`,
            'app/*': `/app/*`,
            '*': `${this.APP_BASE}node_modules/*`
        },
        packages: {
            angular2: {defaultExtension: false}
            // rxjs: {defaultExtension: false}
        },
    };
```
It works after comment out the rxjs related stuff. why is the rxjs without the node_modules prefix?

**Comment by nosachamos** [comment]
@mgechev I'm not sure yet whats happening but @goldenbearkin has a point. The operators you import do work with the default seed configs, but some do not. I had lots of operators imported just like you, but then I tried to import `timeInterval` and it failed. I checked and it exists, path is correct, extension is .js just like the other operators, but it cannot be added until these configs are removed.
